### ISSUE TITLE
try fixing weave issue with plotlyjs.jl

### ISSUE THREAD

**Issue by vjd** [PumasAI/PumasTutorials.jl#97]
[this issue](https://github.com/JunoLab/Weave.jl/issues/47) has been open for a while and since we rely on weave for our tutorials, it will be nice to fix this up. Otherwise, we can move to Publish.jl when ready. Thoughts @MichaelHatherly ?

**Comment by MichaelHatherly** [comment]
So ideal solution here would be to be able to embed the interactive plotly plots into the final document? I'd need to spend some time digging through weave to fully understand whether we can go that route. For now are we just going to be with generating plots via the cairomakie backend?

**Comment by vjd** [comment]
yes, that is what I am trying to do. set `interactive!(false)` and hence ran into https://github.com/PumasAI/PumasPlots.jl/issues/98
